src/conduit-lwt-unix/conduit_lwt_unix.ml

```diff
@@ -410,6 +423,7 @@ let serve ?backlog ?timeout ?stop ~on_exn ~(ctx : ctx) ~(mode : server) callback
 
 let endp_of_flow = function
   | TCP { ip; port; _ } -> `TCP (ip, port)
+  | Tunnel -> `Unknown "TLS tunnel"
```

Comment:
I'm not sure what is the best way to address this? We can't expose `Lwt_io.channel` in `Conduit.endp` obviously, should `conduit-lwt-unix` expose an extended `endp` type with the new tunnel option?

Reply:
I went ahead and exposed a new `Conduit_lwt_unix.endp` which extends `Conduit.endp` with the Lwt-specific TLS tunneling